internal/wgnl/configure_linux.go

```diff
@@ -212,7 +213,7 @@ func sockaddrBytes(endpoint net.UDPAddr) ([]byte, error) {
 
 		sa := unix.RawSockaddrInet6{
 			Family: unix.AF_INET6,
-			Port:   uint16(endpoint.Port),
+			Port:   binary.BigEndian.Uint16(nlenc.Uint16Bytes(uint16(endpoint.Port))),
```

Comment:
Let's split this pattern out into a helper function:

```go
// sockaddrPort interprets port as a big endian uint16 for use passing sockaddr
// structures to the kernel.
func sockaddrPort(port int) uint16 {
    return binary.BigEndian.Uint16(nlenc.Uint16Bytes(uint16(port)))
}
```

Can add it near the bottom of this file.